sentry-core/src/main/java/io/sentry/core/protocol/SentryStackTrace.java

```diff
@@ -7,7 +7,7 @@
 /** The Sentry stacktrace. */
 public class SentryStackTrace implements IUnknownPropertiesConsumer {
   private List<SentryStackFrame> frames;
-  // TODO registers is missing?
+  private Map<String, String> registers;
```

Comment:
@bruno-garcia is it a `Map`?

Reply:
Looks like it: https://docs.sentry.io/development/sdk-dev/event-payloads/stacktrace/#attributes
```
 registers:

    Optional. A map of register names and their values. The values should contain the actual register values of the thread, thus mapping to the last frame in the list.
```